feat(flutter): drift & hive integrations (#8499)
getsentry/sentry-docs: src/platforms/flutter/integrations/hive-instrumentation.mdx

---
title: Hive Database Instrumentation
description: "Learn more about the Sentry Hive Database Instrumentation for the Flutter SDK."
caseStyle: camelCase
supportLevel: production
sdk: sentry.dart.hive
categories:
  - mobile
---

<Include name="feature-stage-beta.mdx" />

_(New in version 7.13.1)_

> The [sentry_hive](https://pub.dev/packages/sentry_hive) package provides `Hive` support for database instrumentation. The source can be found [on GitHub](https://github.com/getsentry/sentry-dart/tree/main/hive). Note, that to capture transactions, you have to first <PlatformLink to="/performance/">set up performance monitoring</PlatformLink>.

Reviewer: ```suggestion
The [sentry_hive](https://pub.dev/packages/sentry_hive) package provides `Hive` support for database instrumentation. The source can be found [on GitHub](https://github.com/getsentry/sentry-dart/tree/main/hive). You must <PlatformLink to="/performance/">Set Up Performance </PlatformLink> monitoring to capture transactions for this integration to work.
```
